### ISSUE TITLE
API Features

### ISSUE THREAD

**Issue by Anthonyg5005** [avarayr/suaveui#6]
I think it'd be a great idea to add support for changing sampling parameters for generation. The models don't seem to be following along too well because sampling is too randomized. It'd also be great to be able to change the prompt template instead of relying on chat completions ad it'd give more control over who is and isn't talking.

**Comment by avarayr** [comment]
Yeah tweaking the generation parameters should be quite easy to add. I'm thinking this will go into the global settings that will apply to all personas.

system prompt should also be able to be customized, but each chat should have the ability to override the default at any moment.

If you're down to make an PR I'd appreciate the help.
